From 8bf57764bd3fab6a78982c8687d4db7edf5c2833 Mon Sep 17 00:00:00 2001
From: Rex Zhu <Rex.Zhu@amd.com>
Date: Thu, 18 Jan 2018 13:31:08 +0800
Subject: [PATCH 4672/5855] drm/amdgpu: Fix build error in gmc_v9_0.c
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

error: ‘struct amdgpu_device’ has no member named ‘mc’
adev->mc.aper_size = adev->mc.real_vram_size;

Change-Id: I5b7b1c9389d74e5c122a8b7261c7224af15a5c51
Reviewed-by: Jim Qu <Jim.Qu@amd.com>
Signed-off-by: Rex Zhu <Rex.Zhu@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
index 543cf66..f67d34d 100644
--- a/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/gmc_v9_0.c
@@ -723,8 +723,8 @@ static int gmc_v9_0_mc_init(struct amdgpu_device *adev)
 
 #ifdef CONFIG_X86_64
 	if (adev->flags & AMD_IS_APU) {
-		adev->mc.aper_base = gfxhub_v1_0_get_mc_fb_offset(adev);
-		adev->mc.aper_size = adev->mc.real_vram_size;
+		adev->gmc.aper_base = gfxhub_v1_0_get_mc_fb_offset(adev);
+		adev->gmc.aper_size = adev->gmc.real_vram_size;
 	}
 #endif
 	/* In case the PCI BAR is larger than the actual amount of vram */
-- 
2.7.4

